#fieldtype1 fieldname1
#
int8 sensor_1
int8 sensor_2
